docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -122,9 +122,37 @@ what should be returned as well as implementing it.  This will
 certainly affect the syncer between mailbox workspace and edge
 cluster, and the summarization part will affect the status summarizer.
 
+### Good handling of workload conflicts
+
+We could start by handling workload conflicts in a very simple way:
+treating each as an error.  Later development can handle them better,
+as outlined [later](#edgeplacement-objects).
+
+### Denaturing/renaturing
+
+We could start by not doing this.  For some resources, the effect of
```

Comment:
I think we probably say something like "do it on a per need basis" or something like that. i.e., We are not going to implement an overall Denaturing/renaturing  , but I also think we will not "do nothing" . I thought we plan to include the KIvorno requirements for this PoC and they have mandatory requirements in this area

Reply:
I thought it was obvious that "we could start by not doing this" implies that we can incrementally progress through doing it for the various resources that need it.  The resources that need denaturing/renaturing are a finite set, so I think we should aim for eventually implementing the whole set.

Reply:
not very clear, but also not very important as I think we will have some of this already implemented for phase 1